tests/bpf2c_tests/elf_bpf.cpp

```diff
@@ -91,7 +91,7 @@ enum class _test_mode
 };
 
 void
-run_test_elf(const std::string& elf_file, _test_mode test_mode)
+run_test_elf(const std::string& elf_file, _test_mode test_mode, const char* type)
```

Comment:
```suggestion
run_test_elf(const std::string& elf_file, _test_mode test_mode, _In_z_ const char* type)
```

Reply:
Changed to using `std::optional<std::string>&`